crates/core/src/store.rs

```diff
@@ -207,11 +208,19 @@ impl StoreBuilder {
             WasiCtxBuilder::Preview1(_) => {
```

Comment:
This isn't new for this PR, but the name of this method and its comment aren't really a reflection of reality. It might make sense to change this method to just `inherit_network`.